uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -350,6 +352,19 @@ define([
       return getAvailableStatusesForCurrentStatus();
     }
 
+    /**
+     * Gets readable type title for the current request
+     *
+     * @return {String}
+     */
+    function getRequestTypeTitle () {
+      return {
+        'leave': 'Leave',
+        'sickness': 'Sickness',
+        'toil': 'TOIL'
+      }[getLeaveType()];
```

Comment:
Can we move these title definitions to each one of the leave request instances (leave, sickness, and toil) ?

Why?

A) It would be easier to get the leave request title: `leaveRequest.title`.
B) It would be easier to remember to add the "title" attribute if a new leave request type is introduced in the future.
C) Other components can take advantage of this.

Reply:
Done.